apps/mail/app/error.tsx

```
@@ -1,29 +1,29 @@
-"use client";
+'use client';
 
-import { Button } from "@/components/ui/button";
-import React, { useEffect } from "react";
+import { Button } from '@/components/ui/button';
+import React, { useEffect } from 'react';
 
 export default function Error({ error, reset }: { error: Error; reset: () => void }) {
```

Biome (1.9.4)

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)